### DISCUSSION TITLE
User to be member of multiple companies

### DISCUSSION THREAD

**Discussion by ShaunNeighbourSwale** [snipe/snipe-it#10291] [Ideas]
Hi

I'm looking at setting up a single Snipe-IT instance to manage ~20 schools.  The ability to setup companies is great, as we don't want most of our schools to be able to see each other's assets.

However, I do have a few staff who work for multiple schools.  They don't need to see all assets, so I don't really want to give them super admin.  equally, there are other staff who only need access to individual schools, so creating a company to cover multiple schools is not really an option.

What would be great is having the ability to grant individual users access to multiple companies.  Is that doable?

Regards

Shaun.

**Comment by Fizpop91** [comment]
Would love this ability/feature too. Being able to assign users to multiple companies would be super useful

**Comment by amkurzhal** [comment]
I would love to have this feature as well. I am in the same position as ShaunNeighborSwale. We actually have some people who manage assets across companies, but should NOT have access to the settings menus. Either multiple companies per user or, at the very least, a global  checkbox that would allow an admin to not be constrained by the Company. Or maybe both to accommodate global technicians and for those people who only need reporting on a few Companies (Schools), but not all.

**Comment by a-carrig** [comment]
I have found a use case where we want to have a global user who can request and get assets from both our companies, but still want to be able to separate inventory and administration of our assets by company.
